src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return RENT * days - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return RENT * days - SHORT_TERM_DISCOUNT;
```

Comment:
The current logic applies a flat discount for rentals of 3 or more days and 7 or more days. If the task requires a different discount structure (e.g., a percentage discount or a different threshold), you should adjust the logic accordingly. Ensure that the discount logic aligns with the task requirements.